simpleclient_hibernate/pom.xml

```diff
@@ -45,7 +45,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
-            <version>4.2.0.Final</version>
+            <version>5.3.1.Final</version>
```

Comment:
5.3.1 was only a bit over a month ago, I doubt many people are using it. What's the oldest version with this feature, and how many people are using at least that version?

Reply:
Changed the dependency to 5.2.0 where the new metric was added (2 years old). If we're to believe https://mvnrepository.com/artifact/org.hibernate/hibernate-core the 5.2.x versions are currently most widely used.